sdk/sdk-api/src/main/java/mil/emp3/api/abstracts/MapService.java

```diff
@@ -21,4 +21,13 @@ protected MapService(String serviceURL)
     public java.net.URL getURL() {
         return this.mapServiceURL;
     }
+
+    @Override
+    public String toString() {
+        if(null != mapServiceURL) {
```

Comment:
No, sub class will print it's own name. MapService is abstract. I updated KMLS toString to call suprt.toString().